streams/src/main/java/org/apache/kafka/streams/kstream/internals/graph/StreamStreamJoinNode.java

```diff
@@ -157,11 +165,6 @@ private StreamStreamJoinNodeBuilder() {
             return this;
         }
 
-        public StreamStreamJoinNodeBuilder<K, V1, V2, VR> withJoined(final Joined<K, V1, V2> joined) {
```

Comment:
This builder method was never called, hence the whole `Joined` field can be removed... But it seems kind of important. Is this right?